doc/api/corepack.md

```diff
@@ -12,10 +12,7 @@ added:
 
 > Stability: 1 - Experimental
 
-Documentation for this tool can be found on the [Corepack repository][].
-
-Despite Corepack being distributed with default installs of Node.js, the package
-managers managed by Corepack are not part of the Node.js distribution, and
-Corepack itself will no longer be distributed with Node.js 25+.
+Corepack is no longer be distributed with Node.js 25+.
+Please refer to [Corepack repository][] for documentation.
```

Comment:
```suggestion
Corepack is no longer distributed with Node.js 25+.
Please refer to the [Corepack repository][] for documentation.
```

Thanks for this PR!

This is just a grammar correction.

I would have the same suggestion as before about describing the versions i.e. "release lines Node.js 25 and above", however since you didn't pick this up in the other PR I don't want to put too much emphasis on it.